internal/characters/heizou/heizou.go

```diff
@@ -33,8 +31,6 @@ func NewChar(s *core.Core, w *character.CharWrapper, _ profile.CharacterProfile)
 	c.SkillCon = 3
 	c.BurstCon = 5
 
-	c.infuseCheckLocation = combat.NewCircleHit(c.Core.Combat.Player(), 0.1, false, combat.TargettableEnemy)
```

Comment:
no change to any of heizou's ability required?

Reply:
The only thing that can absorb is Heizou's Q, but that is implemented as an attack that's centered on primary target, so I don't think I have to change anything there?